### PR TITLE
Update `build.sh`

### PR DESCRIPTION
**Pull request by madsbk**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -115,6 +115,8 @@ if hasArg -v; then
 fi
 if hasArg -g; then
     BUILD_TYPE=Debug
+    export SKBUILD_INSTALL_STRIP=false
+    export SKBUILD_CMAKE_BUILD_TYPE=""Debug""
```
Comment (bdice) [inline build.sh:119]: Why double double quotes?
```suggestion
    export SKBUILD_CMAKE_BUILD_TYPE="Debug"
```

File: docs/source/install.rst [hunk 2]
```diff
@@ -48,11 +48,11 @@ In order to setup a development environment run:
   # CUDA 12.5
   mamba env create --name kvikio-dev --file conda/environments/all_cuda-125_arch-x86_64.yaml
 
-To build and install the extension run:
+The Python library depend on the C++ library, thus we build and install both:
```
Comment (bdice) [inline docs/source/install.rst:51]:
```suggestion
The Python library depends on the C++ library, thus we build and install both:
```

**Review by bdice — APPROVED** [review]
Two comments, please resolve as you see fit. Approving.

File: build.sh (hunk 1 above)
Comment (wence-) [reply build.sh:119]: I just copied as-is from the scikit-build documentation

Comment (madsbk) [reply build.sh]: It looks like a typo, at least other code, including in their own testing, just use `SKBUILD_CMAKE_BUILD_TYPE=Debug`.
Fixed

**Pull request merged**